### ISSUE TITLE
Test migration using did-cli

### ISSUE THREAD

**Issue by olemp** [Puzzlepart/did#917]
### TLDR
Test migration using did-cli 

### Steps

- [x] Create a CosmoDB account _somewhere_ (in puzzlepart dev tenant or own tenant)
- [x] Export all data as CSV from https://portal.azure.com/#@puzzlepart.com/resource/subscriptions/b5e5e285-a57a-4593-a2ef-221dc037ac9f/resourceGroups/pzl-did/providers/Microsoft.Storage/storageAccounts/did85de37e8af91/overview
- [x] Upgrade [did-cli](https://github.com/Puzzlepart/did-cli) to latest version
- [x] Create db `main` in CosmoDB
- [x] did-init with main db and connection string
- [x] Create new subscription using [did-cli](https://github.com/Puzzlepart/did-cli) for Puzzlepart dev or pzlcloud
- [x] Import in the following order: customers, projects, time entries, confirmed periods
- [x] Test logging in locally
